ethereum-consortium-blockchain-network/nested/fiveBootNodeVMExtension.json

```diff
@@ -0,0 +1,131 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "apiVersion" : {
+      "type": "string"
+    },
+    "txNode": {
+      "type": "int"
+    },
+    "bootMNNode": {
+      "type": "int"
+    },
+    "mnNode": {
+      "type": "int"
+    },
+    "txVMNamePrefix": {
+      "type": "string"
+    },
+    "numTXNodes": {
+      "type": "int"
+    },
+    "mnVMNamePrefix": {
+      "type": "string"
+    },
+    "numMNNodes": {
+      "type": "int"
+    },
+    "artifactsLocationURL": {
+      "type": "string"
+    },
+    "adminUsername": {
+      "type": "string"
+    },
+    "ethereumAccountPsswd": {
+      "type": "securestring"
+    },
+    "privateKeyForPrefundedAccount": {
+      "type": "securestring"
+    },
+    "ethereumNetworkID": {
+      "type": "int"
+    },
+    "maxPeers": {
+      "type": "int"
+    },
+    "nodeIDArray": {
+      "type": "array"
+    },
+    "mnNIPrefix": {
+      "type": "string"
+    },
+    "gethIPCPort": {
+      "type": "int"
+    },
+    "nodeKeyArray": {
+      "type": "array"
+    }
+  },
+  "variables": {
+    "numBootNodes": 5
+  },
+  "resources": [
+    {
+      "apiVersion": "[parameters('apiVersion')]",
+      "type": "Microsoft.Compute/virtualMachines/extensions",
+      "name": "[concat(parameters('txVMNamePrefix'), copyIndex(), '/newuserscript')]",
+      "copy": {
+        "name": "txNodesConfigLoop",
+        "count": "[parameters('numTXNodes')]"
+      },
+      "location": "[resourceGroup().location]",
+      "properties": {
+        "publisher": "Microsoft.OSTCExtensions",
+        "type": "CustomScriptForLinux",
+        "typeHandlerVersion": "1.4",
```

Comment:
There is a v2.0 now that is recommended. Needs to be done across all extension templates in this PR.